bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol

```diff
@@ -626,7 +668,10 @@ interface IBondPricing {
         returns (address);
 }
 
-contract HectorBondV2NoTreasuryFTMDepository is OwnableUpgradeable {
+contract HectorBondV2NoTreasuryFTMDepository is
```

Comment:
@bejaxer Will you change the contract name to BondNoTreasury? 
- Rename HectorBondV2NoTreasuryFTMDepository.sol to BondNoTreasury.sol

Reply:
Add a string public variable called version = "2.0"